tools/c7n_gcp/c7n_gcp/filters/recommender.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
How is this file generated? Is it scraping [recommender docs](https://cloud.google.com/recommender/docs/recommenders)? I didn't see an obvious way to list this from the CLI/SDK directly, and don't know if it's something we'd expect [make data-update](https://github.com/cloud-custodian/cloud-custodian/blob/b807094c0f0afd772ce5831276f3fb56da576dfa/Makefile#L132-L139) to cover. (that may be a separate/future issue though...)

Reply:
its beautiful soup, and then hand edits for resource types. so its not clear it would work for make data-update due to the manual modifications, for reference the gist for the script is below. I suppose we could make it respect the hand edits though.

https://gist.github.com/kapilt/376fdb36e59bd48eedd71d36e7602de6